packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -188,6 +199,16 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
         {...positioningContainerProps}
       >
         <div className={classNames.root}>
+          {ariaAlertText && (
+            <div
+              className={classNames.ariaContainer}
+              aria-live="assertive"
```

Comment:
If you used role="alert" will it read? We could then remove the timeout

Reply:
I reverted the change back to a setTimeout and appending a textNode.  It doesn't seem to work unless it's one of these hacky methods:

https://developer.paciellogroup.com/blog/2012/06/html5-accessibility-chops-aria-rolealert-browser-support/